partitura/utils/generic.py

```diff
@@ -171,6 +171,35 @@ def iter_subclasses(cls, _seen=None):
                 yield sub
 
 
+def first_order_derivative(func, x0, dx=0.5):
+    """
+    Compute the first order derivative of a function at a point `x0` using
```

Comment:
mention here where this code was taken from, so people can check it up

Reply:
You're right I will do that